Ladybird/Qt/FindInPageWidget.cpp

```diff
@@ -34,20 +34,23 @@ FindInPageWidget::FindInPageWidget(Tab* tab, WebContentView* content_view)
     m_previous_button = new QPushButton(this);
     m_previous_button->setFixedWidth(30);
     m_previous_button->setIcon(create_tvg_icon_with_theme_colors("up", palette()));
+    m_previous_button->setToolTip("Find the previous match");
```

Comment:
Documentation/HumanInterfaceGuidelines/Text.md says title case for tooltips (and probably omit "the"). Not sure if this applies to the Qt UI bits.